### PR TITLE
Updateios

### PR DESCRIPTION
**Pull request by laszloblum**
android version added

### CODE REVIEW

File: android/src/main/java/de/atroo/securestorage/EncryptedPreferencesImpl.java [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+//                    .setKeyScheme(MasterKey.KeyScheme.AES256_GCM)
+//                    .build();
+//            Log.d(TAG, "init: " + preferencesName);
+//            sharedPreferences = EncryptedSharedPreferences.create(
+//                    context,
+//                    preferencesName,
+//                    masterKey,
+//                    EncryptedSharedPreferences.PrefKeyEncryptionScheme.AES256_SIV,
+//                    EncryptedSharedPreferences.PrefValueEncryptionScheme.AES256_GCM
+//            );
```
Comment (phal0r) [inline android/src/main/java/de/atroo/securestorage/EncryptedPreferencesImpl.java]: @laszloblum WHat about the comment block?

File: android/src/main/java/de/atroo/securestorage/EncryptedPreferencesImpl.java [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+public class EncryptedPreferencesImpl implements SecureStorageInterface {
+    private static final String TAG = "EncryptedPrefsImpl";
+    private SharedPreferences sharedPreferences;
+
+    @Override
+    public boolean init(Context context, String preferencesName) {
+        try {
+            String masterKeyAlias = MasterKeys.getOrCreate(MasterKeys.AES256_GCM_SPEC);
+
+            sharedPreferences = EncryptedSharedPreferences.create(
```
Comment (phal0r) [inline android/src/main/java/de/atroo/securestorage/EncryptedPreferencesImpl.java:24]: @laszloblum The docs (https://developer.android.com/reference/androidx/security/crypto/EncryptedSharedPreferences) mention to specify backup rules for exlcuding the keys. Is this something we should do in the plugin or create a piece of documentation for the usage?

File: android/src/main/java/de/atroo/securestorage/EncryptedPreferencesImpl.java (hunk 1 above)
Comment (laszloblum) [reply android/src/main/java/de/atroo/securestorage/EncryptedPreferencesImpl.java]: Left there unintentionally. Removed

File: android/src/main/java/de/atroo/securestorage/EncryptedPreferencesImpl.java (hunk 2 above)
Comment (laszloblum) [reply android/src/main/java/de/atroo/securestorage/EncryptedPreferencesImpl.java:24]: No, it should be done in the app's AndroidManifest.xml
I have added it to the README.

**Pull request merged**